tutorials/tts/Vits_Training.ipynb

```diff
@@ -0,0 +1,430 @@
+    "\n",
+    "```json\n",
+    "{\"audio_filepath\": \"/path/to/audio1.wav\", \"text\": \"the transcription\", \"duration\": 0.82}\n",
+    "{\"audio_filepath\": \"/path/to/audio2.wav\", \"text\": \"the other transcription\", \"duration\": 2.1}\n",
+    "...\n",
+    "```\n",
+    "Please note that the duration is in seconds.\n",
+    "\n",
+    "\n",
+    "Then you are ready to run your training script:\n",
+    "```bash\n",
+    "python vits.py train_dataset=YOUR_TRAIN.json validation_datasets=YOUR_VAL.json trainer.devices=-1\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evaluating VITS\n",
+    "\n",
+    "Let's evaluate the quality of the VITS model.\n",
+    "\n",
+    "VITS is end-2-end model, so we don't need any additional models to generate audios."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from matplotlib.pyplot import imshow\n",
+    "from matplotlib import pyplot as plt\n",
+    "import IPython.display as ipd\n",
+    "import numpy as np\n",
+    "import torch"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "seed = 1337\n",
+    "\n",
+    "def evaluate_vits_ckpt(model, test):\n",
+    "    with torch.no_grad():\n",
+    "        torch.manual_seed(seed)\n",
+    "        torch.cuda.manual_seed(seed)\n",
+    "        torch.backends.cudnn.enabled = True\n",
+    "        torch.backends.cudnn.benchmark = False\n",
+    "        tokens = model.parse(str_input = test)\n",
+    "        audio = model.convert_text_to_waveform(tokens = tokens)\n",
+    "\n",
+    "\n",
+    "    audio = audio.to('cpu').numpy()\n",
+    "    audio = audio / np.abs(audio).max()\n",
+    "    return audio"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test = \"Hey, this produces speech!\"\n",
+    "\n",
+    "model = VitsModel.from_pretrained(\"tts_en_lj_vits\")\n",
+    "\n",
+    "audio = evaluate_vits_ckpt(model, test)\n",
+    "\n",
+    "print(\"predicted audio\")\n",
+    "ipd.display(ipd.Audio(audio, rate=22050))"
+   ]
+  }
+ ],
+ "metadata": {
+  "accelerator": "GPU",
+  "colab": {
+   "collapsed_sections": [],
+   "name": "Taco2.ipynb",
+   "provenance": []
+  },
+  "interpreter": {
+   "hash": "64f1d17cb7a41c1b8fd5fbc82f0be8f196236a3bf63c75ef58b175e825f8ba9b"
+  },
+  "kernelspec": {
+   "display_name": "Python 3.8.16 ('nemo')",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.16"
```

Comment:
we try to avoid changes like these due to the differences between individual IDE metadata. Could you please clean up your jupyter notebook before updating the PR?